# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../whois/hotfix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For some reason whois fails for some time on T2 (vanilla source)
systems like:

getaddrinfo( ... ): Bad value for ai_flags

Don't use getaddrinfo until this is fully debugged.

  - Rene Rebe <rene@exactcode.de>

--- whois-4.7.26/config.h.vanilla	2008-07-01 20:18:57.000000000 +0200
+++ whois-4.7.26/config.h	2008-07-01 20:19:22.000000000 +0200
@@ -37,7 +37,7 @@
 #if defined __GLIBC__
 # define HAVE_GETOPT_LONG
 # if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1
-#  define HAVE_GETADDRINFO
+#  /*define HAVE_GETADDRINFO */
 # endif
 # if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 7
 #  define HAVE_SHA_CRYPT
